azure-functions/acbs-function/package.json

```diff
@@ -11,6 +11,7 @@
     "axios": "^0.27.2",
     "date-fns": "^2.30.0",
     "dotenv": "^16.3.1",
+    "dtfs": "file:../..",
```

Comment:
Is this required?

Reply:
Ooh these are not, needs to be removed from running the npm install:all on windows, will remove! :)

Reply:
This _should_ stop happening when we move to using npm workspaces :D